src/picongpu/include/plugins/output/sockets/SocketConnector.hpp

```diff
@@ -109,7 +109,9 @@ class SocketConnector
             size_t zipedSize = zip.compress(tmp + MessageHeader::bytes, ((char*) array) + MessageHeader::bytes, size - MessageHeader::bytes, 6);
             MessageHeader* header = (MessageHeader*) tmp;
             header->data.byte = (uint32_t) zipedSize;
-            write(SocketFD, tmp, zipedSize + MessageHeader::bytes);
+            int nbytes = write(SocketFD, tmp, zipedSize + MessageHeader::bytes);
+            if (nbytes < 0)
+                perror("a socket write error occured");
```

Comment:
offline discussed: `perror` is also used in the rest of the class.

Never the less this code is not used anymore therefore we will remove this class in the near future.